_redesign_heros/1.md

```diff
@@ -32,7 +32,7 @@ images:
       </div>
     </div>
     <div class="homepage-hero-banner--new-release-available--version">
-      <a href="/downloads.html">&nbsp;2.13.0 is available!</a>
+      <a href="/downloads.html">&nbsp;1.3.16 is available!</a>
```

Comment:
Please remove this. we don't do much 'advertising' on the 1.x branch aside from a social post. People should know about the latest release (2.x) (edited)